routes/list.js

```diff
@@ -1,9 +1,28 @@
 'use strict';
 
+const fs = require('fs');
+const _ = require('lodash');
+const Handlebars = require('handlebars');
+
 exports.list = {
   path: '/',
   method: 'GET',
   handler(request, reply) {
-    reply('ok');
+    const templateDir = request.server.settings.app.views.path;
+    fs.readdir(templateDir, (err, files) => {
+      if (err) {
+        return reply(err);
+      }
+
+      fs.readFile('files/index.html', 'utf8', (fileErr, contents) => {
```

Comment:
Makes sense. Let's just put the other thing in a method

On Mon, Jun 6, 2016, 4:58 PM Eric Willis <notifications@github.com> wrote:

> In routes/list.js
> <https://github.com/firstandthird/micro-mail/pull/5#discussion_r65990357>:
>
> >  exports.list = {
> >    path: '/',
> >    method: 'GET',
> >    handler(request, reply) {
> > -    reply('ok');
> > +    const templateDir = request.server.settings.app.views.path;
> > +    fs.readdir(templateDir, (err, files) => {
> > +      if (err) {
> > +        return reply(err);
> > +      }
> > +
> > +      fs.readFile('files/index.html', 'utf8', (fileErr, contents) => {
>
> Since this is micro mail - the views folder is really for the templates
> which is a separate thing in our case. This for now is just a "One Off"
> template. @jgallen23 <https://github.com/jgallen23>
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/firstandthird/micro-mail/pull/5/files/5c21d72c3f5a7349a578cbb4cb41eb5b7921f52f#r65990357>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABtIE0rKSjj1Hcp6OFdgwaUoW3v4iv9ks5qJLQQgaJpZM4IvWbu>
> .
>